pkgs/applications/networking/localsend/update.sh

```diff
@@ -16,10 +16,6 @@ fi
 
 sed -i "s/version = \".*\"/version = \"${latestVersion}\"/" "$ROOT/default.nix"
 
-LINUX_x64_URL="https://github.com/localsend/localsend/releases/download/v${latestVersion}/LocalSend-${latestVersion}-linux-x86-64.AppImage"
```

Comment:
Would you be willing to fix updating Linux instead of just removing it?

Reply:
If so, we should also add `passthru.updateScript = ./update.sh;`

Reply:
I don't know how to do that. I would like to have a try but not in this PR.